themes/openy_themes/openy_carnation/src/scss/modules/_footer.scss

```
@@ -124,7 +124,7 @@
     font-size: 15px;
     color: $gray-400;
 
-    #block-openy-carnation-footermenuright {
+    .block-va-ymca-footer-menu-right-menu {
```

Please use not related to the specific project namespace. I recommend something like this
```suggestion
    .block-openy-carnation-footer-menu-right-menu {
```

---

Fixed